src/common.c

```diff
@@ -1403,6 +1407,16 @@ ssize_t ofi_bsock_recvv(struct ofi_bsock *bsock, struct iovec *iov, size_t cnt,
 	return ret ? -ofi_sockerr(): -FI_ENOTCONN;
 }
 
+void ofi_bsock_prefetch_done(struct ofi_bsock *bsock, size_t len)
```

Comment:
I think declaring this, but not using it results in a build warning if using git bisect